source/state_machines_macros/src/to_token_stream.rs

```diff
@@ -276,14 +276,14 @@ pub fn output_primary_stuff(
             if trans.kind == TransitionKind::Init {
                 let args = post_params(&trans.params);
                 rel_fn = quote! {
-                    pub open spec fn #name (#args) -> ::std::primitive::bool {
+                    pub open spec fn #name (#args) -> bool {
```

Comment:
This needs to be a fully qualified path because the user might declare a different `bool` type. I think you can use `core::primitive::bool`.

Reply:
I change to core's bool. But I do not encounter problems with a non-qualified path. Would that cause bad side effect for verification result?

Reply:
Generally speaking macros need to use fully qualified paths in order to be ambiguous. If it's just `bool` then it might conflict with a user-defined type.

I don't think there's a unit test for it though, so I'm not surprised you didn't run into any issues with it.